app/views/admin/editions/_edition_view_edit_buttons.html.erb

```diff
@@ -18,6 +18,35 @@
       <%= content_data_button(@edition) %>
   <% end %>
 </section>
+
+<% if @edition.has_enabled_shareable_preview?(current_user) %>
+  <section>
+    <%
+    utm_uri = preview_document_url_with_auth_bypass_token(@edition)
+    %>
+
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+      <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingOne">
+          <h4 class="panel-title">
+            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+            <i class="glyphicon glyphicon-smaller-than-text glyphicon-chevron-down add-right-margin"></i>
+            Share document preview
+            </a>
+          </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+          <div class="panel-body">
+            <p>Send the preview link to someone so they can see a preview of how the document will appear on GOV.UK. No password or GOV.UK Signon account is needed.</p>
+            <p><strong>Select all and copy the link</strong></p>
+            <textarea class="form-control" rows="3"><%= utm_uri %></textarea>
+          </div>
```

Comment:
I think we should also add the text in here from [Nikin's design](https://www.figma.com/file/f8z3zGgXCEorFKHZVfP6gb/Sharable-previews?node-id=80%3A152) about when the link expires - so "The preview link will expire on [day month year] or when the document is published."

Reply:
We have agreed with Nikin not to include it at this time.